using-mobile-extensions/adobe-analytics-edge/product-variable.md

```diff
@@ -226,53 +168,9 @@ contextData["m.purchase"] = "1"
 
 // send the tracking call - use either a trackAction or trackState call.
 // trackAction example:
-ACPCore.trackAction("purchase", data: contextData)
-// trackState example:
-ACPCore.trackState("Order Confirmation", data: contextData)
-```
-{% endtab %}
-
-{% tab title="React Native" %}
-**JavaScript**
-
-```jsx
-//create a context data dictionary
-var contextData = {};
-
-// add products, a purchase id, a purchase context data key, and any other data you want to collect.
-// Note the special syntax for products
-contextData["&&events"] = "event1";
-contextData["&&products"] = ";Running Shoes;1;69.95,;event1=5.5;eVar1=Merchandising,;Running Socks;10;29.99";
-contextData["m.purchaseid"] = "1234567890";
-contextData["m.purchase"] = "1";
-
-// send the tracking call - use either a trackAction or TrackState call.
-// trackAction example:
-ACPCore.trackAction("purchase", contextData);
-// trackState example:
-ACPCore.trackState("Order Confirmation", contextData);
-```
-{% endtab %}
-
-{% tab title="Flutter" %}
-**Dart**
-
-```dart
-//create a context data dictionary
-var contextData = {};
-
-// add products, a purchase id, a purchase context data key, and any other data you want to collect.
-// Note the special syntax for products
-contextData["&&events"] = "event1";
-contextData["&&products"] = ";Running Shoes;1;69.95,;event1=5.5;eVar1=Merchandising,;Running Socks;10;29.99";
-contextData["m.purchaseid"] = "1234567890";
-contextData["m.purchase"] = "1";
-
-// send the tracking call - use either a trackAction or TrackState call.
-// trackAction example:
-FlutterACPCore.trackAction("purchase", data: contextData);
+MobileCore.track(action: "purchase", data: contextData)
 // trackState example:
-FlutterACPCore.trackState("Order Confirmation", data: contextData);
+MobileCore.track(state: "Order Confirmation", data: contextData)
 ```
 {% endtab %}
 {% endtabs %}
```

Comment:
@imironchuk do we still want to keep this example in here? I would prefer including an example with XDM products instead.

Reply:
@rymorale @praveek  We discussed about this and we decided to have two examples: first one using xdm product and the second one using &&product as described in this doc.

Reply:
i'll work on adding the xdm product example